sdk/core/core-lro/review/core-lro.api.md

```diff
@@ -159,6 +167,15 @@ export interface RawResponse {
     statusCode: number;
 }
 
+// @public
+export interface ResponseBody extends Record<string, unknown> {
+    properties?: {
+        provisioningState?: unknown;
+    } & Record<string, unknown>;
+    provisioningState?: unknown;
+    status?: unknown;
+}
```

Comment:
I would rather not export those models.

Reply:
Actually I just want to export LongRunningOperationLocationHeaders but I am not sure where to put it.

Reply:
The index.ts is updated to export everything from the models module using the star syntax but I think it should export select models only.

Reply:
updated